utilities/llmd_utils.py

```
@@ -0,0 +1,658 @@
+            cmd += " --insecure"
+        else:
+            try:
+                from ocp_resources.resource import get_client  # type: ignore
+
+                client = get_client()
+                ca_bundle = get_ca_bundle(client=client, deployment_mode="raw")
+                if ca_bundle:
+                    cmd += f" --cacert {ca_bundle}"
+                else:
+                    cmd += " --insecure"
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)